packages/driver/cypress/integration/e2e/multi-domain/commands/multi_domain_viewport.spec.ts

```diff
@@ -1,113 +1,143 @@
 // @ts-ignore / session support is needed for visiting about:blank between tests
 context('multi-domain viewport', { experimentalSessionSupport: true }, () => {
-  beforeEach(() => {
+  it('syncs the viewport from the primary to secondary', () => {
```

Comment:
The test changes in here are related to  not being able to call cy.viewport between a click that switches domains in the beforeEach and the switchToDomain command in the test.